spec/presenters/vacancy_presenter_spec.rb

```diff
@@ -112,12 +112,24 @@
       expect(vacancy.working_patterns).to be_nil
     end
 
-    it "returns a working patterns string if working_patterns is set" do
-      vacancy = VacancyPresenter.new(create(:vacancy, working_patterns: %w[full_time part_time]))
-      vacancy.organisation_vacancies.create(organisation: create(:school, name: "Smith High School"))
+    context "when only working_patterns is set" do
+      it "returns a string only containing the working pattern" do
+        vacancy = VacancyPresenter.new(create(:vacancy, working_patterns: %w[full_time part_time], working_patterns_details: nil))
+        vacancy.organisation_vacancies.create(organisation: create(:school, name: "Smith High School"))
+
+        expect(vacancy.show_working_patterns).to eq(I18n.t("jobs.working_patterns_info_many", patterns: "full-time, part-time"))
```

Comment:
I don't think you need to do this unless you feel like it, but it annoys me that we are not using I18n's intelligent counting thing where you can say
```
working_patterns_info:
  one: "Blah blah"
  many: "Blah blah but different"
```
and let it choose which one to use based on the `count` parameter.